src/config/dark-sites.config

```diff
@@ -228,14 +235,17 @@ humblegames.com
 hyper.is
 i3wm.org
 ibookpile.net
+identinator.com
 illicoweb.videotron.com
 imgbox.com
 imgur.com
 indiexpo.net
+infinitezoom.net
```

Comment:
![](https://i.imgur.com/WYW7Wng.png)

Reply:
I don't get that only when js is disabled.

Reply:
![](https://i.imgur.com/4TlMoy4.png)

Reply:
I know that Three.js & Brave doesn't work that well together. Chromium does show it correctly.